core/domain/state_domain.py

```diff
@@ -1401,7 +1413,15 @@ def validate(self):
 
     @classmethod
     def create_default_subtitled_html(cls, content_id):
-        """Create a default SubtitledHtml domain object."""
+        """Create a default SubtitledHtml domain object.
+
+        Args:
+            content_id: str. the id of the content.
+
+        Returns:
+            SubtitledHtml. A default SubtitledHtml domain object, some
+            attribute of that object will be ''.
```

Comment:
some attribute?

Reply:
Hi @aks681,
sorry for the late reply. I missed the email regarding this
Here is the link of the [explanation](https://github.com/oppia/oppia/pull/8504#discussion_r371041359)

Reply:
Hmm, I guess since content_id is filled and there is only one other field, it's fine.